Documentation/Input/Gaze.md

```diff
@@ -0,0 +1,75 @@
+# Gaze
+
+[Gaze](https://docs.microsoft.com/en-us/windows/mixed-reality/gaze) is a form of input that interacts
+with the world based on where the user is looking. Gaze exists in two different flavors
+
+## Head gaze
+
+This type of gaze is based on the direction that the head/camera is looking at. Head gaze is active
+on systems that don't support eye gaze, or in cases where the hardware may support eye gaze, but
+the right set of [permissions and setup](../EyeTracking/EyeTracking_BasicSetup.md#eye-tracking-requirements-checklist)
+has not been performed.
+
+Head gaze is usually associated with HoloLens 1 style interactions involving looking at object by
+placing it in the center of the FOV and then performing the air tap gesture.
+
+## Eye gaze
+
+This type of gaze is based on where the user's eyes are looking. Eye gaze is only present
+on systems that support eye tracking. See the
+[eye tracking documentation](../EyeTracking/EyeTracking_Main.md) for more details on how
+to use eye gaze.
+
+## GazeProvider
+
+Gaze functionality (both head and eye) is provided by the
+[GazeProvider](xref:Microsoft.MixedReality.Toolkit.Input.GazeProvider). This provider can be configured
+in the *Pointer* section of the input system profile:
+
+![Gaze Configuration Entrypoint](../../Documentation/Images/Input/GazeConfigurationEntrypoint.png)
+
+Like other sources of input, the gaze provider interacts with objects in the scene
+through use of a [pointer](), but in the case of the gaze provider, its pointer is an [internal
```

Comment:
are you missing a link to the pointer doc here?

Reply:
Yeah, definitely was. I just got rid of that, because I think the link I was planning on referencing would have interrupted the flow here (i.e. directing folks to the pointer architecture docs would have had them go down the rabbit hole.)